netket/vqs/mc/mc_state/expect_grad.py

```diff
@@ -46,25 +46,31 @@ def expect_and_grad(  # noqa: F811
     use_covariance: TrueT,
     *,
     mutable: Any,
+    **kwargs,
 ) -> Tuple[Stats, PyTree]:
     σ, args = get_local_kernel_arguments(vstate, Ô)
 
     local_estimator_fun = get_local_kernel(vstate, Ô)
 
-    Ō, Ō_grad, new_model_state = grad_expect_hermitian(
+    res = grad_expect_hermitian(
```

Comment:
For internal functions like `grad_expect_hermitian`, wouldn't it be easier to always return the estimators? If the wrapper function doesn't want to return them, their lifetime only extends by a few lines (if the difference isn't jitted away), but it would keep the unpacking simple